keps/sig-node/2411-cri-container-log-rotation/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: CRI Container Log Rotation
+kep-number: 2411
+authors:
+  - "@umohnani8"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-instrumentation
+  - sig-storage
+status: implemented
+creation-date: 2021-02-02
```

Comment:
I suggest you set the creation date to when this initially landed/was discussed. I suspect [this](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/kubelet-cri-logging.md) was the original proposal, and this is just a migration/update. (I'll review the KEP itself later but make sure that's linked!)

Reply:
Set the date to when the original issue was opened https://github.com/kubernetes/kubernetes/issues/58823